src/responsive-fixed-data-table.js

```diff
@@ -62,8 +62,8 @@ export default class ResponsiveFixedDataTable extends React.Component {
 			const { offsetWidth, offsetHeight } = findDOMNode(this);
 
 			this.setState({
-				gridWidth: offsetWidth,
-				gridHeight: offsetHeight
+				gridWidth: offsetWidth || 1,
```

Comment:
I added a constant to set the initial table height. Not sure if this is how you'd prefer to do it.